deps/build.jl

```diff
@@ -4,6 +4,14 @@ using BinDeps
 
 using Compat
 
+# check for cairo version 
+function validate_cairo_version(name,handle)
+    f = Libdl.dlsym_e(handle, "cairo_version")
+    f == C_NULL && return false
+    v = ccall(f, Int32,())        
+    return v > 11000
```

Comment:
Good guestion -> https://www.cairographics.org/manual/index-1.10.html but we don't much of them... 

I have to admit, i just changed the version number to something like all-pass as cairo's earlier than 1.10 are really hard to get. Would you insist on dropping this completely?